packages/caver-klay/caver-klay-accounts/src/index.js

```diff
@@ -109,39 +109,57 @@ Accounts.prototype.create = function create(entropy) {
     return this._addAccountFunctions(Account.create(entropy || utils.randomHex(32)));
 };
 
-Accounts.prototype.privateKeyToAccount = function privateKeyToAccount(privateKey, targetAddressRaw) {
-  var { privateKey: prvKey, address, isHumanReadable } = utils.parsePrivateKey(privateKey)
+// The key parameter can be either normal private key or klaytnWalletKey format.
+Accounts.prototype.privateKeyToAccount = function privateKeyToAccount(key, targetAddressRaw) {
+  let {legacyAccount: account, klaytnWalletKeyAddress} = this.getLegacyAccount(key)
 
-  if (!utils.isValidPrivateKey(prvKey)) throw new Error('Invalid private key')
+  account.address = this.determineAddress(account, klaytnWalletKeyAddress, targetAddressRaw)
+  account.address = account.address.toLowerCase()
+  account.address = utils.addHexPrefix(account.address)
 
-  if (prvKey.slice(0, 2) !== '0x') {
-    prvKey = `0x${prvKey}`
-  }
+  return this._addAccountFunctions(account)
+}
+
+// The key parameter can be either normal private key or klaytnWalletKey format.
```

Comment:
same here.

Reply:
Descriptions are added for each functions :)